### ISSUE TITLE
Campaign stats delivered list has missing emails

### ISSUE THREAD

**Issue by brentkearney** [Mailtrain-org/mailtrain#607]
V2: The statistics for delivered to email addresses is missing an address. URL:

https://mailtrain.mydomain.ca/campaigns/4/statistics/delivered

The delivery count is correct (although the list only had 2 members) at /campaigns/4/statistics, however expanding it shows only 1 email address.

The missing email address previously unsubscribed from the list. That might be related.

**Comment by bures** [comment]
Is this Mailtrain v1 or v2 ?

**Comment by brentkearney** [comment]
v2

I just did another test, sending to a list with 4 members. The delivery count is 6. ;)

Also the Unsubscribed count is 1, but expanding it shows an empty list. Checking the list membership, all 4 members are still subscribed.

**Comment by bures** [comment]
This is pretty weird. Can you send me the log `journalctl -u mailtrain`?

**Comment by brentkearney** [comment]
Sure; I can edit out the addresses. However, I'm using the docker container version. Would it be in the "mailtrain" container? I'll have to install journalctl, and I'm not familiar with alpine linux but can figure it out if I'm on the right path.

**Comment by brentkearney** [comment]
Theres no system logger running in the mailtrain docker container. Do you know where logs are stored?

**Comment by brentkearney** [comment]
I have an idea of what happened with ether delivery count -- I used the same campaign for 2 separate mailouts. The first time went to a list of 2 members. I then edited the campaign to associate it to a list of 4 members, and sent. So the count of 6 deliveries is correct for the total number of deliveries of this campaign. 👍🏼

However, the missing email in the expanded list of delivered addresses, as initially described, is still an issue.
